simulation_parameters/VisualResourceIdentifier.cs

```diff
@@ -16,6 +16,11 @@ public enum VisualResourceIdentifier
 
     CellBurstEffect,
 
+    /// <summary>
+    ///   Iron projectile effect
+    /// </summary>
+    SidenophoreProjectile,
```

Comment:
It's not correct to add values here in the middle, the summary comment on this enum says:
```
Exact values are used in saves so new values must be appended
///   at the end.
```